src/_lib/modules/word-cloud/word-cloud.component.ts

```diff
@@ -65,16 +65,31 @@ export class WordCloudComponent<T extends IWord> implements OnChanges {
          */
         const exportCanvas = this._renderer.createElement('canvas') as HTMLCanvasElement;
         const exportCtx = exportCanvas.getContext('2d');
-        exportCanvas.width = this._canvas.width;
-        exportCanvas.height = this._canvas.height;
+        const ratio = window.devicePixelRatio;
+
+        exportCanvas.width = this._canvas.width * ratio;
+        exportCanvas.height = this._canvas.height * ratio;
+        exportCtx.scale(ratio, ratio);
 
         this._positionedWords.forEach(pw => {
             const point: IPoint = { x: pw.canvasX, y: pw.canvasY };
             this._setFontDetails(exportCtx, pw.fontSize);
             this._drawWord(pw, point, exportCtx);
         });
 
-        return exportCanvas.toDataURL('image/png');
+        const lowestPointOnTheYAxis = Math.min(...this._positionedWords.map(pw => pw.canvasY - pw.height / 2));
```

Comment:
This could generally do with some docs to explain what we're doing and why

Reply:
@garethdn  Let me know if i need to add more info and ill do that

Reply:
Looks good. Couple of typos: heighest -> highest, y-axies -> y-axis

Reply:
opps :D ill update that thanks